### ISSUE TITLE
Errors during 2.4 pre-test with A3A_fnc_createAIcontrols

### ISSUE THREAD

**Issue by Bob-Murphy** [official-antistasi-community/A3-Antistasi#1747]
```
arma3server_2021-02-23_15-20-24 - Test 2 - HC

15:58:03 Error in expression <upX; _pilots pushBack _x} forEach units _groupUAV;
deleteGroup _groupUAV;
};
{[_>
15:58:03   Error position: <_groupUAV;
deleteGroup _groupUAV;
};
{[_>
15:58:03   Error Undefined variable in expression: _groupuav
15:58:03 File functions\CREATE\fn_createAIcontrols.sqf [A3A_fnc_createAIcontrols]..., line 177
```

```
arma3server_2021-02-23_15-20-24 - Test 2 - HC

 2:17:22 Error in expression <Back _uav;
_groupUAV = group (crew _uav select 1);
{[_x] joinSilent _groupX; _pi>
 2:17:22   Error position: <select 1);
{[_x] joinSilent _groupX; _pi>
 2:17:22   Error Zero divisor
 2:17:22 File functions\CREATE\fn_createAIcontrols.sqf [A3A_fnc_createAIcontrols]..., line 176
```

**Comment by jaj22** [comment]
This probably happens if uavsPortable is an empty array or invalid classname for the modset. I don't know which modset would be the problem here, but 3CB seems to lack them generally.

**Comment by jaj22** [comment]
Ok, more likely it's A3A_fnc_createVehicleCrew not working with a UAV input.

**Comment by jaj22** [comment]
Probably fixed by #1797.
